incubator/nodejs-express/image/project/Dockerfile

```diff
@@ -47,4 +47,5 @@ ENV PORT 3000
 USER node
 
 EXPOSE 3000
+# Where is WORKDIR or CWD set?
```

Comment:
Mistake. I'll remove, it was a note to look into a bug that got fixed in nodejs-express and since stacks don't share content easily, I had to clone the fix into the Kafka stack after rebasing.